connector_amazon/models/competitive_price.py

```
@@ -0,0 +1,252 @@
+    our_current_price = fields.Monetary(
+        compute="_compute_our_current_price",
+        help="Our current selling price for this product",
+    )
+
+    _sql_constraints = [
+        (
+            "amz_competitive_price_unique",
+            "unique(product_binding_id, asin, competitive_price_id, fetch_date)",
+            "This competitive price entry already exists.",
+        ),
+    ]
```

Ruff (0.15.2)

[warning] Mutable default value for class attribute

(RUF012)